start-site/src/main/resources/application.yml

```diff
@@ -712,15 +712,25 @@ initializr:
           mappings:
             - compatibilityRange: "[2.0.0.M1,2.1.0.M1)"
               version: 2.22.4
-            - compatibilityRange: "2.1.0.M1"
+            - compatibilityRange: "[2.1.0.M1,2.2.0.M1)"
               version: 2.24.2
-          description: Apache Camel lets you create the Enterprise Integration Patterns to implement routing and mediation rules a Java based Domain Specific Language via Spring.
+          description: Apache Camel is the Swiss knife of integration.
```

Comment:
Please revert this. I am happy to discuss how to update the description but this shouldn't be part as a dependency upgrade.

Reply:
Okay so I can open another PR with just description change?

Reply:
I don't know if we'll change the description to what you've suggested but that discussion should happen independently. Either a dedicated PR or an issue is fine, thank you.

Reply:
Well surely the 3rd party project maintainers should influence what the description is for *their* project

Reply:
I didn't shut it down, did I? Of course, any suggestion to improve the description of an entry is much welcome.

Reply:
Yeah thanks as the one that is currently is a very old description that is out of date and a bit confusing.

How about using what's on the main website

> Camel is an open source integration framework that empowers you to quickly and easily integrate various systems consuming or producing data.

https://camel.apache.org/